<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Biofuel Source Detection System</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnSelectDataset">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Dataset</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDatasetLocation">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>C:\ImagesDataset\</string>
    </property>
   </widget>
   <widget class="QSlider" name="sliderTrainTestSplit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>281</width>
      <height>22</height>
     </rect>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lblTrainTestSplit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Train/Test Split - 50:50</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTrainModel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Train and Evaluate Model</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cBoxSelectModel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>281</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblDataLocation">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>C:Imagestoevaluate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSelectData">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRunModel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Run Model on Data</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTrainTitle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Train Model&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblRunTitle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Run Model&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSelect_Dataset">
   <property name="text">
    <string>Select Training Data</string>
   </property>
  </action>
  <action name="actionSelect_Testing_Data">
   <property name="text">
    <string>Select Testing Data</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
